src/test/java/uk/gov/hmcts/reform/pdf/service/client/PDFServiceClientTest.java

```diff
@@ -0,0 +1,94 @@
+package uk.gov.hmcts.reform.pdf.service.client;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.MediaType;
+import org.springframework.web.client.RestOperations;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public class PDFServiceClientTest {
+
+    private String endpointBase = "http://localhost";
+    private RestOperations restClient = mock(RestOperations.class);
+    private String sampleTemplate = "<html>Test</html>";
+    private PDFServiceClient pdfServiceClient;
+
+    @Before
+    public void setup() {
+        pdfServiceClient = new PDFServiceClient(restClient, URI.create(endpointBase));
+    }
+
+    @Test
+    public void requests_new_endpoint() {
```

Comment:
It's not clear from the test name itself what the test is supposed to be testing.

Reply:
It tests if client sends requests to the new url that ends with `/pdfs`.

Reply:
`void sends_requests_to_then_new_url()` then? :)

Reply:
done